services/ui-src/src/utils/other/mask.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Custom Masks Type Guard
+ * Add any future custom masks here!
+ */
+const customMasks = ["comma-separated"] as const;
+export type CustomMasks = typeof customMasks[number];
+export const isCustomMask = (x: any): x is CustomMasks =>
+  customMasks.includes(x);
+
+/**
+ * Checks if the provided string is contains numbers to mask
+ * @param {String} value
+ * @returns {String}
+ */
+export const isNumberStringMaskable = (value: string): boolean =>
+  value?.match(/\d/) ? true : false;
```

Comment:
I'm confused about what exactly makes a number "maskable". Shouldn't this be the number format regex that allows `0123456789.,` and disallows anything else? 

Also, it seems like this method could just be an inline func. 

Also, the ternary operator is unnecessary here because if it matches, it will return an array and if it does not match, it will return `null`.

Also note the renaming in the suggestion below.

```suggestion
export const isValidNumericalString = (value: string): boolean =>
  value?.match(/\d/)
```